tools/ARIAtools/extractProduct.py

```diff
@@ -1141,7 +1242,9 @@ def export_products(
 
     # If specified, extract ionosphere long wavelength
     ext_corr_lyrs += ['ionosphere']
-    if 'ionosphere' in layers:
+    user_lyrs = list(
```

Comment:
Why can't we use `if 'ionosphere' in layers:` here? Unless I'm mistaken, this yields exactly the same conditional as if `'ionosphere' in layers` but the cognitive load of this `set.intersection(*map(set, [layers, ['ionosphere']]))` expression is very high